src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -402,11 +405,14 @@ public static void ShowAlertDialog(View titleContent, View content, View actionC
             var window = NUIApplication.GetDefaultWindow();
             var defaultNavigator = window.GetDefaultNavigator();
 
-            var dialog = new AlertDialog(titleContent, content, actionContent);
-            SetDialogScrim(dialog);
-
-            var dialogPage = new Page(dialog);
-            defaultNavigator.Push(dialogPage);
+            using (var dialog = new AlertDialog(titleContent, content, actionContent))
```

Comment:
using(){ } should not be used in this case since the constructor of Page does not clone its parameter.